2.4/root/usr/bin/run-mongod

```diff
@@ -114,8 +103,7 @@ else
   # `cleanup` references it.
   (
     unset_env_vars
-    mongod $mongo_common_args --replSet ${MONGODB_REPLICA_NAME} \
-      --keyFile ${MONGODB_KEYFILE_PATH} ${auth_args}
+    mongod $mongo_common_args --replSet ${MONGODB_REPLICA_NAME} ${auth_args}
```

Comment:
I see this was already lacking proper quoting... but maybe it's a good time to add it?

`MONGODB_REPLICA_NAME` should surely be double-quoted, while `mongo_common_args` and `auth_args` we may want them to expand to multiple things... seems weird to have 2 vars that expand to arbitrary things.
Even then, maybe we can quote and treat them as arrays with the magical @ syntax...